packages/py-api/py_api/models/hackathon_participants_models.py

```diff
@@ -21,4 +56,16 @@ class UpdateParticipant(BaseModel):
     email: Optional[EmailStr] = None
     tshirt_size: Optional[TSHIRT_SIZE] = None
     team_name: Optional[str] = None
-    is_verified: Optional[bool] = False
+    is_verified: Optional[bool] = None
```

Comment:
Why this is None? It should be False.

Reply:
To implement the functionality for updating boolean values. If you want to change a bool field back to false you cannot do it if it isnt None. That is why I added the check != None.